src/schema/nodes/figcaption.ts

```diff
@@ -23,7 +23,7 @@ export interface FigCaptionNode extends ManuscriptNode {
 }
 
 export const figcaption: NodeSpec = {
-  content: 'caption_title? caption*',
+  content: 'caption_title caption+',
```

Comment:
We can probably keep those as is, I'm not sure if this will behave correctly on the editor.